proto/greptime/v1/ddl.proto

```diff
@@ -146,6 +147,11 @@ message DropColumns { repeated DropColumn drop_columns = 1; }
 
 message ChangeColumnTypes { repeated ChangeColumnType change_column_types = 1; }
 
+message ChangeFulltext {
+    string column_name = 1;
+    map<string, string> options = 2;
```

Comment:
Can we use struct instead of a hashmap?

```rust
pub struct FulltextOptions {
    pub enable: bool,
    pub analyzer: FulltextAnalyzer,
    pub case_sensitive: bool,
}
```

Reply:
It's compatible with `ColumnOptions`:
```
// Additional options for the column.
message ColumnOptions {
  // Supported keys:
  //   "fulltext":
  //      A JSON encoded string containing full-text search options for the column.
  //
  //      The fulltext options JSON structure:
  //      {
  //        "enable": bool,          // Indicates whether full-text search is
  //                                 // enabled for the column.
  //
  //        "analyzer": string,      // The language-specific text analyzer to
  //                                 // use for indexing and searching text.
  //                                 // Supported values: ["English" (Default), "Chinese"].
  //
  //        "case-sensitive": bool   // Indicates whether the text should be treated
  //                                 // as case-sensitive during full-text search.
  //      }
  //
  //      Example:
  //      "fulltext": "{\"enable\": true, \"analyzer\": \"English\", \"case-sensitive\": false}"
  map<string, string> options = 1;
}
```
Perhaps we can make the changes together during the next pr.

Reply:
> Perhaps we can make the changes together during the next pr.

It sounds good to me. We should always avoid using the hashmap, it makes metadata hard to maintain.